### DISCUSSION TITLE
第2节课笔记(10班)

### DISCUSSION THREAD

**Discussion by isLinXu** [InternLM/Tutorial#116] [笔记]
请在下方提交第 2 节课笔记笔记链接，知乎/CSDN/飞书 等等都可。
笔记让知识记得更牢固💪

**Comment by jiangye520** [comment]
https://mp.weixin.qq.com/s/86nvz3NSDxsI7vg4VeRnvg

**Comment by hannah803** [comment]
https://hannahliu.framer.website/blog/internlm-training-lec-2

**Comment by lishuang70345** [comment]
https://blog.csdn.net/weixin_42748604/article/details/135454562?spm=1001.2014.3001.5502

**Comment by dearSocrates** [comment]
https://blog.csdn.net/rabbit9798/article/details/135436624?spm=1001.2014.3001.5501

**Comment by admlrz** [comment]
https://[blog.csdn.net](https://blog.csdn.net/weixin_44581312/article/details/135466701)/weixin_44581312/article/details/135466701

**Comment by Tian-CASIA** [comment]
https://blog.csdn.net/qq_55090315/article/details/135481699?spm=1001.2014.3001.5502

**Comment by adela778** [comment]
https://blog.csdn.net/Adela0546/article/details/135514452

**Comment by MIracleyin** [comment]
https://nnkwjqimr7.feishu.cn/docx/BlnOd3u4soGLcgxVZkZcnHAtnnf?from=from_copylink
